sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessageSerializer.java

```diff
@@ -341,7 +342,9 @@ private List<ServiceBusReceivedMessage> deserializeListOfMessages(Message amqpMe
         final AmqpResponseCode statusCode = RequestResponseUtils.getStatusCode(amqpMessage);
 
         if (statusCode != AmqpResponseCode.OK) {
-            LOGGER.warning("AMQP response did not contain OK status code. Actual: {}", statusCode);
+            LOGGER.atWarning()
+                .addKeyValue("statusCode", statusCode)
+                .log("AMQP response did not contain OK status code.", statusCode);
```

Comment:
nit; should we need a "{ }" in the format string for _statusCode_ argument?

Reply:
great catch thanks!